themes/digital.gov/src/scss/new/guides/_guide-prev-next.scss

```diff
@@ -2,6 +2,10 @@
 
 .dg-guide__guide-buttons {
```

Comment:
**Todo:** We should rename this in component refactor to improve clarirty because the class name doesn't match the partial name.

Reply:
Added to [card](https://trello.com/c/75yN7Bba/2253-refactor-guide-prev-next-partial).